src/screens/Users/AddOrEditUser.js

```diff
@@ -17,15 +17,17 @@
     with this program; if not, write to the Free Software Foundation, Inc.,
     51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 */
+
 import * as React from 'react';
 import { Platform, TouchableOpacity, View, TouchableWithoutFeedback } from 'react-native';
 import type { NavigationScreenProp } from 'react-navigation';
 import { connect } from 'react-redux';
 import { PERMISSIONS, RESULTS, request as requestPermission } from 'react-native-permissions';
 import styled, { withTheme } from 'styled-components/native';
 import { CachedImage } from 'react-native-cached-image';
-import t from 'tcomb-form-native';
+import tForm from 'tcomb-form-native';
```

Comment:
Could you please check with @wjkg - i believe this was being removed.

Reply:
It wasn't -- I was advised not to use the library in a new component, but there was no removal from existing ones.